src/Infrastructure/Controller/Organization/VisaModel/AddVisaModelController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\Organization\VisaModel;
+
+use App\Application\CommandBusInterface;
+use App\Application\QueryBusInterface;
+use App\Application\VisaModel\Command\SaveVisaModelCommand;
+use App\Infrastructure\Controller\Organization\AbstractOrganizationController;
+use App\Infrastructure\Form\User\VisaModelFormType;
+use App\Infrastructure\Security\Voter\OrganizationVoter;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Requirement\Requirement;
+use Symfony\Component\Routing\RouterInterface;
+
+final class AddVisaModelController extends AbstractOrganizationController
+{
+    public function __construct(
+        private \Twig\Environment $twig,
+        private FormFactoryInterface $formFactory,
+        private RouterInterface $router,
+        private CommandBusInterface $commandBus,
+        QueryBusInterface $queryBus,
+        Security $security,
+    ) {
+        parent::__construct($queryBus, $security);
+    }
+
+    #[Route(
+        '/organizations/{organizationUuid}/visas/add',
```

Comment:
Je viens de le voir en lisant la PR d'edit, utiliser le terme visa_model dans les URLs aussi ?

```suggestion
        '/organizations/{organizationUuid}/visa_models/add',
```